nurseconnect/tasks.py

```diff
@@ -0,0 +1,77 @@
+from __future__ import absolute_import
+
+import json
+import requests
+
+from celery.schedules import crontab
+from celery.task import periodic_task
+
+from django.contrib.auth.models import User
+
+
+URL = "https://praekelt:praekelt@npr-staging.jembi.org:5000/ws/rest/v1/" \
+      "nurseconnectdataset"
```

Comment:
sorry, this is really sloppy, credentials should not be in pull-requests. We'll have to notify Jembi.